test/e2e/validation_test.go

```diff
@@ -56,13 +59,19 @@ func getInstanceID(providerID string) string {
 }
 
 // getInstanceIDsOfNode returns the instanceIDs of all the Windows nodes created
-func getInstanceIDsOfNode(kubeclient kubernetes.Interface) ([]string, error) {
-	nodes, err := kubeclient.CoreV1().Nodes().List(metav1.ListOptions{LabelSelector: wmc.WindowsOSLabel})
+func (t *testContext) getInstanceIDsOfNode() ([]string, error) {
+	// Fill in the nodes to be used.
```

Comment:
I dont understand is this a TODO?

Reply:
I am doing it here - https://github.com/openshift/windows-machine-config-operator/pull/30/files#diff-1a876af15bc4507294c2bc7e5f81dc37R66 and to fill the field I am starting the code here in this line.

Reply:
I still don't follow what you are saying. You are just getting a list of nodes here.